src/VisualStudio/Core/Impl/CodeModel/ExternalElements/AbstractExternalCodeElement.cs

```diff
@@ -56,7 +59,41 @@ protected virtual EnvDTE.vsCMAccess GetAccess()
 
         protected virtual string GetDocComment()
         {
-            throw new NotImplementedException();
+            var symbol = LookupSymbol();
+
+            if (symbol == null)
+            {
+                throw Exceptions.ThrowEFail();
+            }
+
+            var documentationCommentXml = symbol.OriginalDefinition.GetDocumentationCommentXml();
+            if (string.IsNullOrWhiteSpace(documentationCommentXml))
+            {
+                return string.Empty;
+            }
+
+            XElement xml;
+            try
+            {
+                xml = XElement.Parse(documentationCommentXml);
```

Comment:
Why not call Microsoft.CodeAnalysis.Workspaces.DocumentationComment.FromXmlFragment?